Xamarin/MvvmSeed/MvvmSeed/MvvmSeed.Domain/Services/StringRandomizerService.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System;
+using System.Linq;
+using MvvmSeed.Domain.Model;
+
+namespace MvvmSeed.Domain.Services
+{
+    /// <summary>
+    /// Simple implementation of <see cref="IStringRandomizerService"/> that stores randomization info on a local database (Realm)
+    /// </summary>
+    public class StringRandomizerService : IStringRandomizerService
+    {
+        private readonly LocalStorageContext _dbContext;
```

Comment:
> Also: this is a local storage approach where you don't care about being a db, file or remote service?

I'm not sure yet, but chances are this design will assume you have a database, at least initially. I want to keep complexity low.

As for the names, I'll see what can I do there.